geo/map_processing/map_structure.py

```diff
@@ -154,7 +154,10 @@ def s2polygon_from_shapely_polyline(self,
         return line
 
     def plot_cells(self, cells: S2Cell):
-        # create a map
+        '''
+        plot the s2cell covering.
```

Comment:
If you have a single line docstring, just do it like this:

'''plot the s2cell covering'''

But you also need to edit it, e.g.

'''Plot the S2Cell covering.'''

Reply:
done